docs/config/inventory.lst

```diff
@@ -31,17 +37,23 @@
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.pop3.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.redis.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.resmon.elasticsearch.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.resmon.elemental-transcoder.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.resmon.sparkpost.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.resmon.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.smtp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.ssh2.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.tcp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/noit.module.varnish.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/ping_icmp.xml"/>
 <xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/postgres.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/prometheus.xml"/>
+<xi:include xmlns:xi="http://www.w3.org/2001/XInclude" href="modules/rabbitmq_driver.xml"/>
```

Comment:
This is a stratcond module and is already included in that section, so it is not needed here.

Reply:
done